### PR TITLE
Feat/get all companies

### PR DESCRIPTION
**Pull request by MoinMarcell**

### CODE REVIEW

**Review by devBK84 — APPROVED** [review]
Deine PR könnten auch kleiner sein... :)

File: backend/pom.xml [hunk 1]
```diff
@@ -48,6 +48,11 @@
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-actuator</artifactId>
+		</dependency>
+
```
Comment (Flooooooooooorian) [inline backend/pom.xml:55]: For what do you need this?

File: frontend/src/components/CompanyCard.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                        <li>{customer.firstname}</li>
+                    )
+                })}
+            </ul>
+            <ul>
+                {props.company.products.map(product => {
+                    return(
+                        <li>{product.name}</li>
+                    )
+                })}
```
Comment (Flooooooooooorian) [inline frontend/src/components/CompanyCard.tsx:32]: You need to add the key prop when you map a list to an jsx list

File: frontend/src/components/CompanyGallery.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+type CompanyGalleryProps = {
+    companies: Company[]
+}
+
+export default function CompanyGallery(props: CompanyGalleryProps){
+    const companyCard = props.companies.map(company => {
+        return(
+            <CompanyCard company={company} />
+        )
+    })
```
Comment (Flooooooooooorian) [inline frontend/src/components/CompanyGallery.tsx:13]: Add key prop

**Review by Flooooooooooorian — APPROVED** [review]
LGTM, fix your code smells

File: backend/pom.xml (hunk 1 above)
Comment (MoinMarcell) [reply backend/pom.xml:55]: Intellij said I need this ^^

**Pull request merged**